packages/replay/test/unit/index-integrationSettings.test.ts

```diff
@@ -1,108 +1,106 @@
 import { mockSdk } from '@test';
 
-import { Replay } from '../../src';
-
-let replay: Replay;
-
 beforeEach(() => {
   jest.resetModules();
 });
 
-describe('blockAllMedia', () => {
-  it('sets the correct configuration when `blockAllMedia` is disabled', async () => {
-    ({ replay } = await mockSdk({ replayOptions: { blockAllMedia: false } }));
+describe('integration settings', () => {
+  describe('blockAllMedia', () => {
+    it('sets the correct configuration when `blockAllMedia` is disabled', async () => {
+      const { replay } = await mockSdk({ replayOptions: { blockAllMedia: false } });
```

Comment:
Good change (never knew that the previous syntax actually works 🤔 )

Reply:
yeah, we should prob. do that everywhere! But just stumbled over this because of some other refactoring made necessary here 😅